lib/jekyll/drops/site_drop.rb

```diff
@@ -37,6 +37,10 @@ def collections
         @site_collections ||= @obj.collections.values.sort_by(&:label).map(&:to_liquid)
       end
 
+      def plugins
```

Comment:
@ashmaroli 's comment made me think about that. Maybe we should call this something like `enabled_plugins` instead? Otherwise this might be considered a breaking change as `{{ site.plugins }}` before this PR was referencing the `plugins` config key.